src/move/econia/sources/market.move

```diff
@@ -2566,14 +2551,23 @@ module econia::market {
                 E_POST_OR_ABORT_CROSSES_SPREAD);
         // Calculate base asset amount corresponding to size in lots.
         let base = (size as u128) * (order_book_ref_mut.lot_size as u128);
+        if (base > (HI_64 as u128)) {
+            base = (HI_64 as u128);
+        };
         // Assert corresponding base asset amount fits in a u64.
         assert!(base <= (HI_64 as u128), E_SIZE_BASE_OVERFLOW);
         // Calculate tick amount corresponding to size in lots.
         let ticks = (size as u128) * (price as u128);
+        if (ticks > (HI_64 as u128)) {
+            ticks = (HI_64 as u128);
+        };
         // Assert corresponding tick amount fits in a u64.
         assert!(ticks <= (HI_64 as u128), E_SIZE_PRICE_TICKS_OVERFLOW);
         // Calculate amount of quote required to fill size at price.
         let quote = ticks * (order_book_ref_mut.tick_size as u128);
+        if (quote > (HI_64 as u128)) {
+            quote = (HI_64 as u128);
+        };
```

Comment:
These should get covered by the range checks below. I advocate the minimum refactor possible and I don't think we need these.

@bogberry your thoughts?